administrator/components/com_admin/sql/updates/postgresql/4.0.0-2021-05-10.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX "#__finder_taxonomy_idx_level" on "#__finder_taxonomy" ("level");
```

Comment:
```suggestion
CREATE INDEX "#__finder_taxonomy_level" on "#__finder_taxonomy" ("level");
```
The index name in the update SQL script has to be the same as in the extension.sql for new installations, otherwise you get an endless loop of DB errors fo be fixed in the database checker.